_sass/_color-matrix.scss

```diff
@@ -0,0 +1,170 @@
+// "Marge" is a cross between USWDS's "Medium" and "Large" breakpoints.
+// It's the equivalent of Bootstrap's "Medium" size.
+$marge-screen: 992px;
+
+// The WAI-ARIA specification states the following about aria-hidden:
+//
+//   Authors MAY, with caution, use aria-hidden to hide visibly rendered
+//   content from assistive technologies only if the act of hiding this
+//   content is intended to improve the experience for users of assistive
+//   technologies by removing redundant or extraneous content. Authors
+//   using aria-hidden to hide visible content from screen readers MUST
+//   ensure that identical or equivalent meaning and functionality is
+//   exposed to assistive technologies.
+//
+// Unfortunately, however, the U.S. Web Design Standards forcibly set
+// anything with aria-hidden to display: none, making it difficult to
+// address the above use case. The following rule allows us to override
+// this default and create content that is visible to sighted users but
+// irrelevant to non-sighted users.
+.usa-sr-invisible[aria-hidden="true"] {
```

Comment:
So I gave this a bit of thought and my inclination is to commit this as-is, but add a new issue to upgrade to the new uswds and remove the `.usa-sr-invisible` class. That way we can deal with any possible regressions in a separate PR and keep this one nicely scoped to just the color matrix.